addons/mail/wizard/mail_compose_message.py

```diff
@@ -379,6 +376,35 @@ def _inverse_reply_to_mode(self):
         for composer in self:
             composer.reply_to_force_new = composer.reply_to_mode == 'new'
 
+    @api.depends('composition_mode', 'model', 'parent_id', 'res_ids', 'template_id')
+    def _compute_partner_ids(self):
+        """ Computation is coming either from template, either from context.
+        When having a template it uses its 3 fields 'email_cc', 'email_to' and
+        'partner_to', in monorecord comment mode. Emails are converted into
+        partners, creating new ones when the email does not match any existing
+        partner. Composer does not deal with emails but only with partners.
+        When having a template in other modes, no recipients are computed
+        as it is done at sending time. When removing the template, reset it.
+
+        When not having a template, recipients may come from the parent in
+        comment mode, to be sure to notify the same people. """
+        for composer in self:
+            if (composer.template_id and composer.composition_mode == 'comment'
+                and not composer.composition_batch):
+                res_ids = composer._evaluate_res_ids() or [0]
+                rendered_values = composer._generate_template_for_composer(
+                    composer.template_id,
+                    res_ids,
+                    {'email_cc', 'email_to', 'partner_ids'},
+                    find_or_create_partners=True,
```

Comment:
Mmmh, seems very unusual to have a computed value that creates records.

So if you open a composer, switch between a couple templates, then close it, it will create partners?

Usually this kind of computation is located in create/write overrides to avoid issues.